.ci_support/win_64_cuda_compilernvcccuda_compiler_version11.8python3.8.____cpython.yaml

```diff
@@ -17,11 +17,11 @@ m2w64_c_compiler:
 m2w64_cxx_compiler:
 - m2w64-toolchain
 pin_run_as_build:
-  r-base:
+  python:
     min_pin: x.x
     max_pin: x.x
-r_base:
-- '4.1'
+python:
+- 3.8.* *_cpython
 target_platform:
 - win-64
```

Comment:
This is the Windows CUDA 11.8 file. It should be removed by the `skip`, but it is not for some reason. Will need to take a closer look

Note part of the issue here is even if the build doesn't create any packages simply downloading the CTK can take over 20mins. So will want to make sure this job is properly removed so it isn't wasting CI cycles

Reply:
@jakirkham I added back `noarch: python` and re-rendered the recipe. It looks like our replacement for `noarch: python` somehow broke the selector logic. Once I added back `noarch: python` to `meta.yaml`, the build for Windows CUDA 11.8 is finally removed.